packages/util/src/signature.ts

```diff
@@ -128,6 +143,7 @@ export const fromRpcSig = function (sig: string): ECDSASignature {
     throw new Error('Invalid signature length')
   }
 
+  const recovery: bigint = calculateSigRecoveryFromV(v)
```

Comment:
Small nit: you do not need to explicity type this, this is inferred by the method call.